src/analysis/souffle/may_will.dl

```diff
@@ -37,8 +37,23 @@ disallowedUsage(dataConsumer, usage, owner, tag) :-
 .decl allUsagesAllowed(dummy_var: symbol)
 
 .decl isUsage(usage: Usage)
+// DO NOT SUBMIT
```

Comment:
It seems like there's some chance that this comment was meant to remind you not to submit this commit? Maybe it means something else, though. I thought I'd bring it up either way.

Reply:
Good catch, I thought I got all of these. I put, effectively, debug prints by making some relations `.output`. I marked those with `// DO NOT SUBMIT`. I'll remove this.

Reply:
Done.